proposed/version-support.md

```diff
@@ -0,0 +1,111 @@
+an approximate schedule for when they will adopt and drop support for PHP versions.
+By synchronizing release schedules, no project is overly penalized for being first
+to drop support for a given PHP version and public hosts have a clear timeline
+they can factor into their normal upgrade schedules.
+
+## Definitions
+
+* **Major version** - A major version of a project is a release that is documented
+as containing API breaks from a previous release.  Typically a major version 
+also contains new features or functionality but that is not required.
+
+* **Minor version** - A minor version of a project is a release that is documented
+as containing new features or functionality but not API breaks.  Typically a minor 
+version also contains bug fixes but that is not required.
+
+* **Patch version** - A patch version of a project is a release that is documented
+as containing bug fixes but no new features or API changes.
+
+* **Support** - A given release of a project *supports* a given PHP release if
+it executes without parse errors and performs its designated task as expected.
+It is not required to leverage any particular language feature or syntax, only
+that the code executes without error.
+
+* **Not Support** - A given release of a project that *does not support* a given
+PHP release if no effort is make to ensure that it executes without parse errors
+or performs as expected.  While the code may execute correctly, that is considered
+to be incidental.  A project SHOULD NOT attempt to break compatibility with a given
+PHP version simply for its own sake; it should only not consider that PHP version
+either way when determining what language features to use.
+
+* **Offer** - A public web host *offers* a PHP version if its customers are able
+to host their application using that PHP version on all hosting plans that offer
+PHP, regardless of cost.  A web host MAY require a user to select a non-default
+option in a control panel or similar, but MUST NOT require a custom configuration
+or other special arrangements with the host.  This requirement applies to both
+new and existing accounts.
+
+* **Participating Project** - A *participating project* is a PHP project that
+openly and publicly declares its intent to comply with the expectations of this 
+standard.  A participating project MUST make reasonable effort to comply with
+the expectations of this standard.
+
+* **Participating Host** - A *participating host* is a company that offers web
+hosting to customers that includes the ability to execute PHP code and openly and
+publicly declares its intent to comply with the expectations of this standard. A
+participating host MUST make reasonable effort to comply with the expecatations of
+this standard.
+
+Such hosts MAY be specific to a particular project or projects, or MAY offer
+general hosting for arbitrary PHP code. Such hosts MAY offer "shared hosting",
+virtual machines, dedicated hardware, cloud configurations, or any other form of
```

Comment:
For VMs and dedicated hardware is this not irrelevant as the management of the php version lays in the hands of the client?